charts/etcd/templates/etcd-compaction-cronjob.yaml

```diff
@@ -50,7 +50,8 @@ spec:
             command:
             - etcdbrctl
             - compact
-            - --data-dir=/var/etcd/data/new.etcd
+            - --data-dir=/var/etcd/data
+            - --snapstore-temp-directory=/var/etcd/data
```

Comment:
The compact subcommand implicitly appends `compactor-XXX` to the `--data-dir`, so it would get it's own folder anyway.

I will make the changes for `--snapstore-temp-directory` though, to make sure all snapshots have a separate dir

Reply:
Updated